flent/gui.py

```diff
@@ -1031,7 +1031,7 @@ def update_progress(self):
         if (p, s) == (0, 0):
             if not self.aborted:
                 elapsed = time.time() - self.start_time
-                self.progressBar.setValue(100 * elapsed / self.total_time)
+                self.progressBar.setValue(100 * elapsed // self.total_time)
```

Comment:
This still fails for me. Integer division can return a float if one or both of the variables are floats, so this should probably be changed to:

```suggestion
                self.progressBar.setValue(int(100 * elapsed // self.total_time))
```

Reply:
Or would this be equivalent?

```suggestion
                self.progressBar.setValue(int(100 * elapsed / self.total_time))
```